src/components/utils.js

```diff
@@ -159,7 +143,15 @@ export function useGridItemSize() {
       PubSub.unsubscribe(gridResizeToken);
       window.removeEventListener('resize', onResizeDebounced);
     };
-  }, []);
+  }, [selector]);
 
   return [width, height, containerRef];
 }
+
+export function pluralize(word, count = null) {
+  return `${word}${(count === 1 ? '' : 's')}`;
+}
```

Comment:
Maybe we know now that we're only applying it to a few words, but we don't know how it will be used down the road. Preference just not to get into pluralization, but if we do, probably better to use library.

Reply:
I have removed this logic and added new parameters for overriding plural axis/title/label/tooltip values.